### PR TITLE
4 test cases with GCONPROD + item 7 equal to WELL

### PR DESCRIPTION
**Pull request by tskille**
Item 7 in GCONPROD equal to WELL is currently not supported, but hopefully this can come in near future

### CODE REVIEW

File: gconprod/T1W.DATA [hunk 1]
```diff
@@ -0,0 +1,286 @@
+-- This reservoir simulation deck is made available under the Open Database
+-- License: http://opendatacommons.org/licenses/odbl/1.0/. Any rights in
+-- individual contents of the database are licensed under the Database Contents
+-- License: http://opendatacommons.org/licenses/dbcl/1.0/
+
+-- Copyright (C) 2024 Equinor
+
+-- This deck is testing keyword GCONPROD + Item 7 = WELL. Well with maximum water production
+-- in group C1 should close when liquid production from group C1 exceeds 7500.
```
Comment (bska) [inline gconprod/T1W.DATA:9]: Tiny nit, this says water production exceeds 7500, but the `GCONPROD` keyword [implies water production exceeds 750](https://github.com/OPM/opm-tests/blob/2cecbe1c250d2f03860150f10a30ccc278080454/gconprod/T1W.DATA#L255).  Is it worthwhile to update the comment to match the actual limit?

**Review by bska — COMMENTED** [review]
This looks good to me except for one possible misprint in the problem description of `T1W.DATA`.

File: gconprod/T1W.DATA (hunk 1 above)
Comment (tskille) [reply gconprod/T1W.DATA]: Thanks Bård, I really appreciate your finding these errors. Should be fixed now

Comment (bska) [reply gconprod/T1W.DATA]: > Should be fixed now

Confirmed.  Thanks a lot for the update!

**Pull request merged**